### PR TITLE
Added Burrow

### PR DESCRIPTION
**Pull request by Xiaro**
Big exploit

### CODE REVIEW

**Comment by Sahil-108** [comment]
Rename to burrow plz

File: src/main/java/com/gamesense/client/module/modules/exploits/AutoCrash.java [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+import com.gamesense.client.module.Module;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+@Module.Declaration(
+    name = "AutoCrash",
```
[the remark this answers is not in the document]
Comment (Xiaro) [reply src/main/java/com/gamesense/client/module/modules/exploits/AutoCrash.java:14]: Ok

**Comment by notperry1234567890** [comment]
yes

**Comment by Doogie13** [comment]
please merge this right now

**Comment by notperry1234567890** [comment]
insane please add

**Comment by tbdontop** [comment]
cant wait for it to be added

**Review by tbdontop — APPROVED** [review]

**Comment by Sahil-108** [comment]
yea idk why hoosier aint merging this

**Comment by olliem5** [comment]
amazing code, why is this not merged??

**Comment by lukflug** [comment]
Time to merge this!

**Pull request merged**